docs/Userguide_sharing_data.rst

```diff
@@ -0,0 +1,101 @@
+Sharing Data with ACLs
+======================
+
+As an illustrative example, to allow ``$USER`` to share with ``$USER2`` in
+``rwx`` a hierarchy ``/network/scratch/$USER/X/Y/Z/...`` with ACLs:
+
+----
+
+| Grant **oneself** permissions to access any **future** files/folders created
+  by the other *(or oneself)*
+| (``-d`` renders this permission a "default" / inheritable one)
+
+.. code-block:: bash
+
+   setfacl -Rdm user:$USER:rwx  /network/scratch/$USER/X/Y/Z/
+
+----
+
+| Grant **another** permission to access any **future** files/folders created
+  by the other *(or oneself)*
+| (``-d`` renders this permission a "default" / inheritable one)
+
+.. code-block:: bash
+
+   setfacl -Rdm user:$USER2:rwx /network/scratch/$USER/X/Y/Z/
+
+----
+
+| Grant **another** permission to access any **existing** files/folders created
+  by *oneself*.
+| Such files and folders were created before the new default ACLs were added
+  above and thus did not inherit them from their parent folder at the moment of
+  their creation.
+
+.. code-block:: bash
+
+   setfacl -Rm  user:$USER2:rwx /network/scratch/$USER/X/Y/Z/
+
+----
+
+| Grant **another** permission to search through one's hierarchy down to the
+  shared location in question.
+
+* **Non**-recursive (!!!!)
+* May also grant ``:rx`` in unlikely event others listing your folders on the
+  path is not troublesome or desirable.
+
+----
+
+| In order to access a file, all folders from the root (``/``) down to the
```

Comment:
Is this part of the previous section on "searching"? It _looks_ like it's its own different example, but the content seems like it's part of the previous block